### PR TITLE
Video updates

### PR DESCRIPTION
**Pull request by jason-cf**
Adding new videos. Making the AI image generator playground series into a video series style like DO. Updating the stream component. Removing ReactJS from Pages and into tutorials list until a static assets page is created. Adding NextJS to the static assets frameworks page.

### CODE REVIEW

File: src/content/docs/workers-ai/tutorials/image-generation-playground/image-generator-flux-newmodels.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  - Typescript
+prev: true
+
+---
+
+import { Details, DirectoryListing, Stream } from "~/components"
+
+In part 2, Kristian expands upon the existing environment built in part 1, by showing you how to integrate new AI models and introduce new parameters that allow you to customize how images are generated. 
+
+<Stream videoId="167ba3a7a86f966650f3315e6cb02e0d" videoTitle="Add New AI Models to your Playground (Part 2)" thumbnailTimeOrURL="2.5s" moreVideosLink="false" />
```
Comment (Oxyjun) [inline src/content/docs/workers-ai/tutorials/image-generation-playground/image-generator-flux-newmodels.mdx]:
```suggestion
<Stream videoId="167ba3a7a86f966650f3315e6cb02e0d" videoTitle="Add New AI Models to Your Playground (Part 2)" thumbnailTimeOrURL="2.5s" moreVideosLink="false" />
```
Making it all caps for consistency.

File: src/content/docs/workers-ai/tutorials/image-generation-playground/image-generator-flux-newmodels.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Refer to the AI Image Playground [GitHub Repository](https://github.com/kristianfreeman/workers-ai-image-playground) to follow along locally.
```
Comment (Oxyjun) [inline src/content/docs/workers-ai/tutorials/image-generation-playground/image-generator-flux-newmodels.mdx]:
```suggestion
Refer to the AI Image Playground [GitHub repository](https://github.com/kristianfreeman/workers-ai-image-playground) to follow along locally.
```

File: src/content/docs/workers-ai/tutorials/image-generation-playground/image-generator-flux.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  - Workers
+languages:
+  - Typescript
+next: true
+
+---
+
+import { Details, DirectoryListing, Stream } from "~/components"
+
+The new flux models on Workers AI are our most powerful text to image AI models yet. In this video, we show you how to deploy your own Workers AI image playground in just a few minutes.
```
Comment (Oxyjun) [inline src/content/docs/workers-ai/tutorials/image-generation-playground/image-generator-flux.mdx]:
```suggestion
The new flux models on Workers AI are our most powerful text-to-image AI models yet. In this video, we show you how to deploy your own Workers AI Image Playground in just a few minutes.
```

File: src/content/docs/workers-ai/tutorials/image-generation-playground/image-generator-flux.mdx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+There are many businesses being built on top of AI image generation models and using Workers AI, you can get access to the best models in the industry without having to worry about inference, ops, or deployment. We provide the API for AI image generation and in a couple of seconds get an image back.
```
Comment (Oxyjun) [inline src/content/docs/workers-ai/tutorials/image-generation-playground/image-generator-flux.mdx]:
```suggestion
There are many businesses being built on top of AI image generation models. Using Workers AI, you can get access to the best models in the industry without having to worry about inference, ops, or deployment. We provide the API for AI image generation, and in a couple of seconds get an image back.
```

File: src/content/docs/workers-ai/tutorials/image-generation-playground/image-generator-flux.mdx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+<Stream videoId="aeafae151e84a81be19c52c2348e9bab" videoTitle="Build an AI Image Generator Playground (Part 1)" thumbnailTimeOrURL="12s" moreVideosLink="false" />
+
+Refer to the AI Image Playground [GitHub Repository](https://github.com/kristianfreeman/workers-ai-image-playground) to follow along locally.
```
Comment (Oxyjun) [inline src/content/docs/workers-ai/tutorials/image-generation-playground/image-generator-flux.mdx]:
```suggestion
Refer to the AI Image Playground [GitHub repository](https://github.com/kristianfreeman/workers-ai-image-playground) to follow along locally.
```

File: src/content/docs/workers-ai/tutorials/image-generator-flux.mdx [hunk 6]
```diff
@@ -14,10 +14,12 @@ languages:
...
-import { Stream } from "~/components"
+import { Details, DirectoryListing, Stream } from "~/components"
 
-The new flux models on Workers AI are our most powerful text to image AI models yet. In this video, we show you how to deploy your own Workers AI image playground in just a few minutes.
+In this series of videos, Kristian Freeman builds an AI Image Playground. To get started, click on part 1 below.
 
-There are many businesses being built on top of AI image generation models and using Workers AI, you can get access to the best models in the industry without having to worry about inference, ops, or deployment. We provide the API for AI image generation and in a couple of seconds get an image back.
+<Details header="Video Series" open>
 
-<Stream videoId="aeafae151e84a81be19c52c2348e9bab" videoTitle="How to build an AI Image Generator using Cloudflare Workers AI" thumbnailTimeOrURL="1.95s" />
```
Comment (Oxyjun) [inline src/content/docs/workers-ai/tutorials/image-generator-flux.mdx:23]: I think there's a slight mix when it comes to capitalisations for video titles - I don't have a preference either way as long as its consistent. Not sure if there's existing guideline for this?

File: src/content/docs/workers/frameworks/framework-guides/nextjs.mdx [hunk 7]
```diff
@@ -32,6 +33,10 @@ To create a new Next.js app, pre-configured to run on Cloudflare using [`@openne
 	}}
 />
 
+## Video Tutorial
+
+<Stream videoId="0b28fdd5938c4929bd7fdedcd167044d" videoTitle="Deploy NextJS to your Workers Application" thumbnailTimeOrURL="2.5s" />
```
Comment (Oxyjun) [inline src/content/docs/workers/frameworks/framework-guides/nextjs.mdx]:
```suggestion
<Stream videoId="0b28fdd5938c4929bd7fdedcd167044d" videoTitle="Deploy NextJS to Your Workers Application" thumbnailTimeOrURL="2.5s" />
```
Or alternatively, "Deploy NextJS to your Workers application"

File: src/content/docs/workers/runtime-apis/rpc/index.mdx [hunk 8]
```diff
@@ -230,6 +230,12 @@ In this way, the introducer Worker can connect two Workers that did not otherwis
 
 Currently, this proxying only lasts until the end of the Workers' execution contexts. A proxy connection cannot be persisted for later use.
 
+## Video Tutorial
+
+In this video, we explore how Cloudflare Workers supports Remote Procedure Calls (RPC) to simplify communication between Workers. Learn how to implement RPC in your JavaScript applications and build serverless solutions with ease. Whether you're managing microservices or optimizing web architecture, this tutorial will show you how to quickly set up and use Cloudflare Workers for RPC calls. By the end of this video, you'll understand how to call functions between Workers, pass functions as arguments, and implement user authentication with Cloudflare Workers.
```
Comment (Oxyjun) [inline src/content/docs/workers/runtime-apis/rpc/index.mdx]:
```suggestion
In this video, we explore how Cloudflare Workers support Remote Procedure Calls (RPC) to simplify communication between Workers. Learn how to implement RPC in your JavaScript applications and build serverless solutions with ease. Whether you're managing microservices or optimizing web architecture, this tutorial will show you how to quickly set up and use Cloudflare Workers for RPC calls. By the end of this video, you'll understand how to call functions between Workers, pass functions as arguments, and implement user authentication with Cloudflare Workers.
```

**Review by Oxyjun — APPROVED** [review]
Some minor suggestions to consider :D

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**